rbac/migration_tool/utils.py

```diff
@@ -91,6 +90,15 @@ def stringify_spicedb_relationship(rel: common_pb2.Relationship):
     )
 
 
+def relationship_to_json(rel):
```

Comment:
Could we add
```
from google.protobuf.json_format import MessageToJson

json_rel = MessageToJson(rel)
```
(e.g. https://stackoverflow.com/questions/19734617/protobuf-to-json-in-python)
Instead?

This would standardise going to and from json.